From c604da7a0fe27cdd9a2728f68d592d13d48f7d46 Mon Sep 17 00:00:00 2001
From: Wentao Zhang <wentao.zhang@windriver.com>
Date: Fri, 26 Aug 2022 09:00:25 +0800
Subject: [PATCH 2/2] Revert upstream commit a8a310c5e6

Debian doesn't have package dh-sequence-package-notes.
Revert upstream commit as it breaks the package builds.

Signed-off-by: Wentao Zhang<Wentao.Zhang@windriver.com>
---
 debian/control | 1 -
 debian/rules   | 1 -
 2 files changed, 2 deletions(-)

diff --git a/debian/control b/debian/control
index ddd464b..1cfee09 100644
--- a/debian/control
+++ b/debian/control
@@ -14,7 +14,6 @@ Vcs-Git: https://salsa.debian.org/systemd-team/systemd.git
 Vcs-Browser: https://salsa.debian.org/systemd-team/systemd
 Homepage: https://www.freedesktop.org/wiki/Software/systemd
 Build-Depends: debhelper-compat (= 13),
-               dh-sequence-package-notes,
                pkg-config,
                xsltproc <!nodoc>,
                docbook-xsl <!nodoc>,
diff --git a/debian/rules b/debian/rules
index f65291a..f962f4e 100755
--- a/debian/rules
+++ b/debian/rules
@@ -6,7 +6,6 @@
 export LC_ALL = C.UTF-8
 
 include /usr/share/dpkg/default.mk
-include /usr/share/debhelper/dh_package_notes/package-notes.mk
 
 ifeq ($(DEB_VENDOR),Ubuntu)
     DEFAULT_NTP_SERVERS = ntp.ubuntu.com
-- 
2.25.1

